src/api/expenses/expenses.controller.js

```diff
@@ -0,0 +1,87 @@
+const { expensesService } = require('../../services/expenses/expenses.service');
+const { usersService } = require('../../services/users/users.service');
+const { filterExpensesWithQuery } = require('../../services/helperService');
+const getAll = (req, res) => {
+  res.send(filterExpensesWithQuery(req.query));
+};
+
+const create = (req, res) => {
+  const { userId, spentAt, title, amount, category, note } = req.body;
+  const user = usersService.getById(userId ?? 0);
+
+  if (!userId || !spentAt || !title || !amount || !category || !user) {
+    res.sendStatus(400);
+
+    return;
+  }
+
+  const expense = expensesService.create({
+    userId,
+    spentAt,
+    title,
+    amount,
+    category,
+    note,
+  });
+
+  res.status(201).send(expense);
```

Comment:
```suggestion
  res.status(201).send(expense);
} catch(err) {
  res.status(500).send('Please try later')
}
```

Reply:
error catch added as we don't want server to go down.